tool/docker/ubuntu/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM ubuntu:17.10
+
+MAINTAINER chris.seaton@oracle.com
+
+# We need a conventional locale for testing
+ENV LANG=en_US.UTF-8
+
+RUN apt-get update
+
+# To clone source repositories
+RUN apt-get install -y git=1:2.11.0-4 mercurial=3.9.1-1
+
+# To bootstrap our own JVMCI-comptaible JDK we need a JDK
+RUN apt-get install -y openjdk-8-jdk=8u131-b11-2ubuntu1 openjdk-8-source=8u131-b11-2ubuntu1
+
+# Other dependencies for building a JDK
+RUN apt-get install -y make=4.1-9.1 gcc=4:6.3.0-2ubuntu1 g++=4:6.3.0-2ubuntu1
+
+# To build Sulong and TruffleRuby's C extensions
+RUN apt-get install -y clang=1:4.0-36ubuntu3 llvm-dev=1:4.0-36ubuntu3 libc++-dev=3.9.1-2 libc++abi-dev=3.9.1-2
```

Comment:
No, we'd have to build it ourselves I believe. I'm trying to not do that where possible.